lib/Document/Editable/EditableHandler.php

```diff
@@ -149,6 +149,7 @@ public function __construct(
     }
 
     /**
+     * @deprecated 
```

Comment:
```suggestion
     * @deprecated Will be removed in Pimcore 11
```

... and on the other ones also?